resources/dicts/events/death/beach.json

```diff
@@ -174,16 +190,30 @@
         "weight": 20,
         "event_text": "m_c drowned after being pushed out to sea by o_c warriors.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": [
+                "adolescent",
+                "young adult",
+                "adult",
+                "senior adult" 
+                ],
+            "status": [
```

Comment:
you didn't list leader, but you did include a lead_death.  i do think it would be good to include leads and deps in this (and maybe some of the other ranks like mediator and med cat?